### DISCUSSION TITLE
git update and not work published 0 from metadata

### DISCUSSION THREAD

**Discussion by jarx88** [requarks/wiki#5723] [Error / Bug Report] [answered]
hi

I use obsidian to create new page in metadata i put
```
---
title: sdfsdfsdf
description: fsdfsdfsdf
published: 0
date: 2022-10-04T18:22:56.456Z
tags: 
editor: markdown
dateCreated: 2022-10-04T18:22:56.456Z
---
```

When I post this note to Github and I click Force Sync in the Wiki. The page shows up to me but is published, it should be non-public with parametr published: **0**

Is Bug or am I doing something wrong ?

**Comment by NGPixel** [comment, marked as answer]
There's a mismatch in the output and expect input.

While it outputs `published`, the input expects `isPublished` (the internal property name). While you could use `isPublished`, it will be overwritten back to `published` if you make an edit from wiki.js.

A future update will fix this.
